src/main/scala/nl.knaw.dans.easy.bag2deposit/Command.scala

```diff
@@ -40,6 +40,7 @@ object Command extends App with DebugEnhancedLogging {
       load((cfgPath / "application.properties").toJava)
     }
   }
+  val fedoraProvider = FedoraProvider(properties)
```

Comment:
Please move down after bagParentDirs (it could otherwise hurt --help or --verion attempts) and make it private.

Reply:
done